### ISSUE TITLE
Following a <Link /> results in DOMException: Failed to execute 'removeChild' on 'Node'

### ISSUE THREAD

**Issue by craig95** [theolampert/react-flickity-component#28]
If we follow a `<Link />` from [React Router](https://knowbody.github.io/react-router-docs/api/Link.html) the view to which the app is routed is not rendered. The HTML body is empty. 

The full error message is: 
```
Uncaught DOMException: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
```

**Comment by yaodingyd** [comment]
Could you provide a sandbox link?

**Comment by craig95** [comment]
I made a [sandbox](https://codesandbox.io/s/0ppj3r5k30). I cannot replicate the error because I am getting another error.

**Comment by danielmahon** [comment]
@craig95 @yaodingyd See my fix in #29
